bin/evaluate.sh

```diff
@@ -4,4 +4,11 @@ set -o errexit
 
 TERM=xterm rustc - -o ./out "$@"
 printf '\377' # 255 in octal
-exec ./out
+if [ "${*#*--test}" != "$*" ] && [ "${*#*--color=always}" != "$*" ]; then
+	# For /evaluate.json, we have {test: true, color: true}. Let's make the output coloured too.
+	# This would be better in web.py, but we don't have an easy way to alot parameters for ./out.
+	# Really, these shell scripts shouldn't exist at all.
```

Comment:
@alexcrichton Personally I think it *does* help as it explains why a dubious special case is as it is, including also mentioning how things should be instead. I can remove it if you really want, but I do not believe that it should be removed.